### PR TITLE
WIP/TST: troubleshoot xfails

### PR DESCRIPTION
**Pull request by jbrockmendel**
- [ ] closes #xxxx
- [ ] tests added / passed
- [ ] Ensure all linting tests pass, see [here](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#pre-commit) for how to run them
- [ ] whatsnew entry

### CODE REVIEW

**Comment by jreback** [comment]
pls rebase

**Comment by jbrockmendel** [comment]
will keep rebasing, but i don't expect this to be mergeable any time soon

File: pandas/tests/generic/test_duplicate_labels.py [hunk 1]
```diff
@@ -94,7 +94,6 @@ def test_preserve_getitem(self):
         assert df.loc[[0]].flags.allows_duplicate_labels is False
         assert df.loc[0, ["A"]].flags.allows_duplicate_labels is False
 
-    @pytest.mark.xfail(reason="Unclear behavior.")
```
Comment (jbrockmendel) [inline pandas/tests/generic/test_duplicate_labels.py:97]: @TomAugspurger IIRC you were the driving force behind `flags`.  do you expect to finish these off in the not-too-distant future?

Comment (TomAugspurger) [reply pandas/tests/generic/test_duplicate_labels.py]: No plans right now.

**Comment by jreback** [comment]
one more rebase pls

**Comment by jbrockmendel** [comment]
> one more rebase pls

will keep rebasing, but pls dont merge; even once green i want to re-run a handful of times to try to catch flaky failures

**Comment by jreback** [comment]
got it can u change the title to WIP or make a draft

**Comment by jreback** [comment]
prob should rebase this

**Comment by jreback** [comment]
status here?

**Comment by jbrockmendel** [comment]
> status here?

The OK-looking changes i've been breaking off into the TODO/FIXME PRs.  Mostly hoping to keep running until the pytest-timeout catches something useful.

**Pull request closed without merging**